docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -56,6 +48,44 @@ Python style
           """
           pass
 
+Imports
+-------
+
+* Use convenience imports whenever available. For example, do this::
+
+      from django.views.generic import View
+
+  Don't do this::
+
+      from django.views.generic.base import View
+
+* Put imports in five groups: future, standard library, third-party libraries,
+  other Django components, the same component; sort lines in each group
+  alphabetically by the full module name. Place ``import module`` statements
```

Comment:
Do we really place ``import ...`` after ``from .. import ...``? I think we mix them right now and just sort them by module.